### DISCUSSION TITLE
Feature Request - Ability to Transfer Issues from Private Repos to Public Repos, with a warning.

### DISCUSSION THREAD

**Discussion by brettpac** [community/community#37636] [Projects and Issues]
Nice thread about it here....
https://github.com/orgs/community/discussions/21979

Please, please, please, please, please, pleeeeeeeeeeaaaaaaaaaaaaaaaaaaaase!

**Comment by ossanna16** [comment]
Hi there @brettpac and welcome to our community! Thank you for the feedback!
